Dockerfile

```diff
@@ -38,7 +38,6 @@ RUN rm -rf /var/lib/apt/lists/*
 RUN git clone https://github.com/CityOfZion/neo-python.git /neo-python
 WORKDIR /neo-python
 # RUN git checkout development
-RUN git checkout v0.8.2
```

Comment:
Is the intention still to checkout development as the comment implies? If so I recommend replacing with `RUN git checkout development`

Reply:
I think I see from https://github.com/CityOfZion/neo-privatenet-docker/issues/64#issue-400713932 that the matter branch is desired. If so, why leave the commented code in the released product?

Reply:
This is basically reverting https://github.com/CityOfZion/neo-privatenet-docker/commit/708f6d3a400f112da24f80e4aa5b6c7fc3b63b9e